device-types/HPE/Aruba-EC-S-P-AC.yaml

```diff
@@ -1,43 +1,68 @@
 ---
 manufacturer: HPE
-model: Aruba ec-s-p-ac
+model: Aruba EC-S-P AC
 slug: hpe-aruba-ec-s-p-ac
 part_number: '201687'
 u_height: 1
 is_full_depth: false
 airflow: front-to-rear
+weight: 18.14
+weight_unit: lb
+front_image: true
+rear_image: true
 console-ports:
   - name: console
     type: rj-45
     label: console
 power-ports:
```

Comment:
Hi Thank you for your contribution.
Are these modular PSUs? If so they should be modeled as `module-bays` with the required PSUs added to the modules folder if they don't already exist.